src/transformers/models/bart/modeling_bart.py

```diff
@@ -502,7 +502,7 @@ def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
         return hidden_states
 
 
-class BartPretrainedModel(PreTrainedModel):
+class BartPreTrainedModel(PreTrainedModel):
```

Comment:
Switching to the proper name with capital T, and creating an alias for backward compatibility.